backend/fms_core/migrations/0067_v4_12_0.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 4.2.4 on 2024-10-17 17:30
```

Comment:
Do not create a new migration file ... use the existing one that is not for index...

Reply:
removed